### ISSUE TITLE
Missing include

### ISSUE THREAD

**Issue by clapas** [screenplaydev/plutil#5]
Hello,

I had to add the following in the `Libraries/plist/Sources/Format/Encoding.cpp` file:
```
include <cstdlib>
```
in order to be able to compile the sources. This library provides the `abort()` function.

Cheers

**Comment by Neverbolt** [comment]
Yup, still an issue when I tried to build it. Thanks @clapas for the fix!
